Lib/test/support/__init__.py

```diff
@@ -804,20 +804,19 @@ def check_cflags_pgo():
     return any(option in cflags_nodist for option in pgo_options)
 
 
-_header = 'nP'
-_align = '0n'
+_header = '2P'
```

Comment:
This should not be changed

```suggestion
_header = 'nP'
```